pkg/providers/amifamily/ami.go

```diff
@@ -94,45 +97,80 @@ func (p *Provider) KubeServerVersion(ctx context.Context) (string, error) {
 	return version, nil
 }
 
-// Get returns a set of AMIIDs and corresponding instance types. AMI may vary due to architecture, accelerator, etc
+// MapInstanceTypes returns a set of AMIIDs and corresponding instance types. AMI may vary due to architecture, accelerator, etc
```

Comment:
nit: Update this comment so that it more accurately reflects what the function is doing